source/initial_temperature/ascii_data_layered.cc

```diff
@@ -0,0 +1,117 @@
+/*
+  Copyright (C) 2011 - 2019 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+ */
+
+
+#include <aspect/global.h>
+#include <aspect/initial_temperature/ascii_data_layered.h>
+
+namespace aspect
+{
+  namespace InitialTemperature
+  {
+    template <int dim>
+    AsciiDataLayered<dim>::AsciiDataLayered ()
+    {}
```

Comment:
can the constructor be deleted?

Reply:
I don't think so? At least, when I try, I get a bunch of `error: type ‘aspect::Utilities::AsciiDataLayered<2>’ is not a base type for type ‘aspect::InitialTemperature::AsciiDataLayered<2>’` errors